### ISSUE TITLE
<title>

### ISSUE THREAD

**Issue by kcpevey** [nebari-dev/jupyterlab-new-launcher#40]
### Context

When you click "notebook" via "create empty", the modal has a lot of space for "start a new kernel" and very little space for "connect to a running kernel". The "Connect to a running kernel" table has only one line visible. 

https://github.com/nebari-dev/jupyterlab-new-launcher/assets/15659110/1bd08259-2395-434a-ad70-00eb6ba56e76

### Value and/or benefit

More balanced UI and improved UX when selecting a "running kernel" 

### Anything else?

Sidenote - I dont know what "connect to a running kernel" means. Instead of launching new, do I want to just load an already running notebook??

**Comment by krassowski** [comment]
>  I dont know what "connect to a running kernel" means. Instead of launching new, do I want to just load an already running notebook?

Yes, this is useful if you want to draft some code in a console side-by-side with notebook:
- one user flow is to have notebook on the left for noting down the results and console on the right for prototyping code; once a user is happy with a result they drag the cell out of console to the notebook - this way all previous attempts are still tracked, but do not clobber the notebook.
- another user story is to have one notebook for live data visualisation, and another for the actual model code, open side-by-side connected to the same kernel; this allows you to monitor the results of your models without scrolling back and forth.

**Comment by krassowski** [comment]
While the top table takes up more space, I cannot quite reproduce the bottom table getting down to a single row, neither in Chrome nor in Firefox:

| Chrome | Firefox |
|--|--|
| ![image](https://github.com/nebari-dev/jupyterlab-new-launcher/assets/5832902/20ffbd44-7d6e-4022-8952-96043008735a) | ![image](https://github.com/nebari-dev/jupyterlab-new-launcher/assets/5832902/d15e70fe-0ff7-485e-814f-b749cf5c892c) |

I suspect this state can arise due to delayed rendering with react.

**Comment by kcpevey** [comment]
Now I'm in a state where the bottom table is reduced down to a tiny sliver. The top table has a scroll bar, but its still taking up too much room. 

![image](https://github.com/user-attachments/assets/efb24f55-173f-429d-bf01-2b3cdf6ce5bc)

**Comment by krassowski** [comment]
Reproduced in Chrome. I had to create at least 40 environments and resize browser window down.

![image](https://github.com/user-attachments/assets/be59f9fa-aa3c-4023-ad50-a2660b59588b)
